keys/localhost.pem

```diff
@@ -1,31 +1,46 @@
 -----BEGIN CERTIFICATE-----
```

Comment:
When running from docker OS was complaining that the certificate is too short, that's why I updated it.

Reply:
We could switch to ssl_context = 'adhoc' , this will generate a certificate for each app start instead. Got the tip from @anestos this morning. :)